src/Raven.Client/Documents/TcpUtils.cs

```diff
@@ -70,5 +69,12 @@ internal static class TcpUtils
             stream = sslStream;
             return stream;
         }
+
+        public static string GetTcpUrl(string host)
+        {
+            if (host.IndexOf(".fiddler", StringComparison.OrdinalIgnoreCase) > 0)
```

Comment:
What if the user will supply a localhost.fiddler url in order to debug RavenDB?
In this case all tcp connections won't work.